astropy/coordinates/tests/test_geodetic_representations.py

```diff
@@ -2,49 +2,116 @@
 
 """Test geodetic representations"""
 import pytest
-from numpy.testing import assert_array_equal
 
 from astropy import units as u
-from astropy.coordinates.earth import (
+from astropy.coordinates.geodetic import (
+    BaseGeodeticRepresentation,
     GRS80GeodeticRepresentation,
     WGS72GeodeticRepresentation,
     WGS84GeodeticRepresentation,
 )
-from astropy.coordinates.representation import CartesianRepresentation
+
+from astropy.coordinates.representation import (
+    CartesianRepresentation,
+    REPRESENTATION_CLASSES,
+)
 from astropy.units import allclose as quantity_allclose
+from astropy.units.tests.test_quantity_erfa_ufuncs import vvd
+
+# Preserve the original REPRESENTATION_CLASSES dict so that importing
+#   the test file doesn't add a persistent test subclass (LogDRepresentation)
+from astropy.coordinates.tests.test_representation import (
+    setup_function,
+    teardown_function,
+)
+
+
+def test_custombasegeodetic_error():
+    # Test incomplete initialization.
+
+    msg = "module 'erfa' has no attribute 'foo'"
+    with pytest.raises(AttributeError, match=msg):
 
+        class CustomGeodeticEllipsoidError(BaseGeodeticRepresentation):
```

Comment:
If I read the code I can figure out why an error is implemented as a subclass of `BaseGeodeticRepresentation`, but I'd rather not spend the effort.
```suggestion
        class InvalidCustomGeodeticEllipsoid(BaseGeodeticRepresentation):
```
This also applies to the other class in this function.